src/ts/models/UserModel.ts

```diff
@@ -70,17 +70,17 @@ export class UserModel {
     /**
      * The city the user lives in
      */
-    public readonly city: string;
+    public readonly city: string | null;
```

Comment:
Syntax to be consistent would be:
city?: string

which declares a nullable string